packages/commonwealth/client/scripts/views/pages/search/helpers.tsx

```diff
@@ -132,9 +132,9 @@ const getCommunityResult = (community, setRoute) => {
 
   const onSelect = () => {
     if (params.community) {
-      setRoute(params.community.id ? `/${params.community.id}` : '/');
+      setRoute(params.community.id ? `/${params.community.id}` : '/', {}, null);
```

Comment:
FFR: Would be good to have some context here as a comment, so anyone working on this code in the future will understand this is for cross-chain search results, passing null instead of a prefix. @martins87